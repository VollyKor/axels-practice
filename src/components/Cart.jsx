import { Row, Col, Container } from "react-bootstrap";

import { ShippingForm, OrderNav, OrderSummary } from "components";

export default function Cart() {
  return (
    <Container>
      <Col lg={9} className="mx-auto ">
        <Row className="mx-auto bg-white shadow rounded">
          <Col lg={7} className="px-4">
            <OrderNav />
            <ShippingForm />
          </Col>
          <Col lg={5} className="position-relative bg-light rounded">
            <OrderSummary />
          </Col>
        </Row>
      </Col>
    </Container>
  );
}
