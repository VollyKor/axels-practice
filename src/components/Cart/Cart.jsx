import ShippingForm from "../ShippingForm/ShippingForm";
import OrderNav from "../OrderNav";
import OrderSummary from "../OrderSummary/OrderSummary";
import { Row, Col, Container } from "react-bootstrap";

export default function Cart() {
  return (
    <Container>
      <Col lg={9} className="mx-auto ">
        <Row className="mx-auto bg-white shadow rounded">
          <Col lg={7} className="px-4">
            <OrderNav />
            <ShippingForm />
          </Col>
          <Col lg={5} className="position-relative bg-light rounded">
            <OrderSummary />
          </Col>
        </Row>
      </Col>
    </Container>
  );
}
