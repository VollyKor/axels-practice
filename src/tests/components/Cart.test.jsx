import { shallow } from 'enzyme';

import Cart from 'components/Cart';

it('renders snapshot', () => {
    const wrapper = shallow(<Cart />);

    expect(wrapper).toMatchSnapshot();
});
