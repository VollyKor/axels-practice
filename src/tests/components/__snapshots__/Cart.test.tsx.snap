// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot 1`] = `
<Container
  fluid={false}
>
  <Col
    className="mx-auto"
    lg={9}
  >
    <Styled(Row)
      className="mx-auto bg-white shadow rounded"
    >
      <Col
        className="px-4"
        lg={7}
      >
        <Route
          path="/cart"
        >
          <OrderForms />
        </Route>
      </Col>
      <Col
        className="position-relative bg-light rounded"
        lg={5}
      >
        <OrderSummary />
        <Route
          path="/cart/success"
        >
          <styled.div />
        </Route>
      </Col>
    </Styled(Row)>
  </Col>
</Container>
`;
